picard/ui/forms/ui_options_metadata.py

```diff
@@ -1,6 +1,6 @@
 # Form implementation generated from reading ui file 'ui/options_metadata.ui'
 #
-# Created by: PyQt6 UI code generator 6.6.1
+# Created by: PyQt6 UI code generator 6.9.0
```

Comment:
Running `python setup.py build_ui` changed a couple of other `ui_*.py` files as well, but I ignored them as the only change was this version number.
The file `picard/resources.py` has been completely rewritten, but that seems to be caused by a different PyQt version on my machine rather than by changes in this PR?